backend/python/app/tests/services/test_story_service.py

```diff
@@ -0,0 +1,124 @@
+from ...models.story import Story
+
+
+# https://github.com/apryor6/flask_testing_examples/tree/master/fte
+def test_add_story(app, db):  # noqa
+    # This is not a very useful test, as it basically is testing SqlAlchemy; however,
+    # I leave it here as a reference for how to create objects
+    obj = Story(title="title", description="description", youtube_link="", level=1)
+    db.session.add(obj)
+    db.session.commit()
+    fetched = Story.query.all()
+    assert len(fetched) == 1
+    assert fetched[0].title == "title"
+
+
+def assert_story_equals_model(story_response, story_model):
```

Comment:
@gaoxk 
> would prefer if this is at the top of the file, bottom of the file, or in a util file.
> could also be a function written into the Story model! like to_dict

@e-wai 
> Think this is a similar conversation to the test_story_mutation tests and how we want to test models? But also yes, how do we want to handle helpers?

Reply:
by helpers do you mean functions like `assert_story_equals_model`? something like a file `app/tests/utils.py` could work, or the functions could be baked into models

Reply:
Created a separate directory for helpers, `app/tests/helpers` for eg. `app/tests/helpers/story_helper.py`. Do you think there will be more general helpers that wouldn't belong to one of these?